app/plugins/ConductrPlugin.scala

```diff
@@ -0,0 +1,20 @@
+package plugins
+
+import com.typesafe.conductr.bundlelib.play.StatusService
+import com.typesafe.conductr.bundlelib.play.ConnectionContext.Implicits
+import play.api.{ Application, Plugin, Logger }
+import play.api.libs.concurrent.Execution
+
+class ConductrPlugin extends Plugin {
+  
+  override def onStart() {
+    import Implicits.defaultContext
+    StatusService.signalStartedOrExit()
+    Logger.info("Application has started")
```

Comment:
Thanks Rich. Seems inconsistent to me though by not having objects like that injectable.

> On 3 Apr 2015, at 11:01 pm, Rich Dougherty <notifications@github.com> wrote:
> 
> In app/plugins/ConductrPlugin.scala:
> 
> > @@ -0,0 +1,20 @@
> > +package plugins
> > +
> > +import com.typesafe.conductr.bundlelib.play.StatusService
> > +import com.typesafe.conductr.bundlelib.play.ConnectionContext.Implicits
> > +import play.api.{ Application, Plugin, Logger }
> > +import play.api.libs.concurrent.Execution
> > +
> > +class ConductrPlugin extends Plugin {
> > +  
> > +  override def onStart() {
> > +    import Implicits.defaultContext
> > +    StatusService.signalStartedOrExit()
> > +    Logger.info("Application has started")
> No Logger doesn't seem to be injectable. I think we decided it was so basic that it wasn't useful, but people could bind/inject the underlying Logback objects if they want. (James probably remembers more if you want to check with him.)
> 
> —
> Reply to this email directly or view it on GitHub.
>